src/coremedia.cs

```diff
@@ -196,10 +207,16 @@ interface CMSampleBufferAttachmentSettings {
 
 		[iOS (11,0), Mac (10,13), TV (11,0)]
 		NSData CameraIntrinsicMatrix { get; set; }
+
+		[iOS (13,0), Mac (10,15), TV (13,0), Watch (6,0)]
+		nint AudioIndependentSampleDecoderRefreshCount { get; set; }
+
+		[Mac (10,10), Watch (6,0)]
+		bool ForceKeyFrame { get; set; }
```

Comment:
same, new but without iOS/tvOS availability

Reply:
Same reason as the key :)